Xamarin.Forms.Platform.Android/Extensions/NativeBindingExtensions.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using Android.Widget;
+using AView = Android.Views.View;
+
+namespace Xamarin.Forms.Platform.Android
+{
+	public static class NativeBindingExtensions
+	{
+		public static void SetBinding(this AView self, Expression<Func<object>> memberLamda, Binding binding)
```

Comment:
I would not add `SetBinding()` overrides taking `Expression` or `lambda`. At least for now. The main usage for those is to be refactor-compliant, but `nameof()` do that JustFine™